docs/release-notes/NuGet-6.3.md

```diff
@@ -0,0 +1,125 @@
+
+* [Feature] Error on multiple entries for the same package ids in Directory.Packages.props - [#9467](https://github.com/NuGet/Home/issues/9467)
+
+### Issues fixed in this release
+
+**DCRs:**
+
+* [DCR]: Only cancel VS cred provider requests if VS is closing - [#11970](https://github.com/NuGet/Home/issues/11970)
+
+* Remove net45 and net40 targets from the NuGet build - [#11830](https://github.com/NuGet/Home/issues/11830)
+
+* For C++/CLI PackageReference projects, NuGet should ignore the TargetPlatformMoniker - [#11808](https://github.com/NuGet/Home/issues/11808)
+
+* [DCR]: Include caught exceptions as inner exceptions when rethrowing (in MsBuildUtility) - [#11766](https://github.com/NuGet/Home/issues/11766)
+
+* [DCR]: Print sources in NU1507 - [#11715](https://github.com/NuGet/Home/issues/11715)
+
+* Specifying both -f ... and -r ... to dotnet build fails to restore if multiple frameworks are present in the project file - [#11653](https://github.com/NuGet/Home/issues/11653)
+
+* PackageSourceMapping public constructor - [#11609](https://github.com/NuGet/Home/issues/11609)
+
+* Add opt-out escape hatch on Linux and macOS - [#11264](https://github.com/NuGet/Home/issues/11264)
+
+* Add support for system and fallback certificate bundles - [#11263](https://github.com/NuGet/Home/issues/11263)
+
+* Duplicate PackageReference includes are handled inconsistently by all restores, leading to unnecessary full restores - [#9864](https://github.com/NuGet/Home/issues/9864)
+
+**Bugs:**
+
+* [Bug]:  X.509 trust store isn't initialized in `dotnet add package` and SDK resolver code paths - [#11956](https://github.com/NuGet/Home/issues/11956)
+
+* Cache DTE service in VS Solution Manager - [#11902](https://github.com/NuGet/Home/issues/11902)
+
+* Nuget CPS references reader is forcing all vc projects to be fully loaded - [#11877](https://github.com/NuGet/Home/issues/11877)
+
+* Make dotnet package verification env var value comparison case insensitive - [#11876](https://github.com/NuGet/Home/issues/11876)
+
+* Using JsonTextWriter manually in LockFileFormat - [#11870](https://github.com/NuGet/Home/issues/11870)
+
+* Extra allocations in EqualityUtility - [#11867](https://github.com/NuGet/Home/issues/11867)
+
+* [Bug]: Boxing of structs to compute hashcode is causing excessive allocations - [#11866](https://github.com/NuGet/Home/issues/11866)
+
+* When restore raises an NU1301, build might fail with a `project.assets.json doesn't have a target for 'net6.0-windows10.0.19041.0` like error that's a red herring - [#11862](https://github.com/NuGet/Home/issues/11862)
+
+* [Bug]: Package source option "All" appears unsorted in the in the list when using VS in non-English languages - [#11857](https://github.com/NuGet/Home/issues/11857)
+
+* [Bug]: Installing a package in Visual Studio when using central package management results in an error before projects can be modified - [#11828](https://github.com/NuGet/Home/issues/11828)
```

Comment:
This is truly a bug, but we should be calling it out in the important issues. 

In particular maybe we can say

`Package installation in Visual Studio respects Central Package Management`. 

I think it'd be great if @JonDouglas and @jeffkl can help clarify and wordsmith.

Reply:
I think that's a good start. This bug blocked the experience, so perhaps we leave this as is until we announce project system support in 17.4?

Reply:
It would probably be fine to add something to the What's New section saying "When using Central Package Management, Visual Studio no longer errors when installing packages and instead the project and central package management file are updated".

Reply:
I'm adding this to "What's New" and removing the `[Bug]` tag